### ISSUE TITLE
Tag imports from Omeka

### ISSUE THREAD

**Issue by omegader** [scholarslab/Neatline#311]
It would be helpful to have the option to import tags when adding Omeka items, just as Title, Coverage, etc. are automatically mapped onto Neatline equivalents.

It might also help to be able to import other fields from Omeka. For example, a database with columns containing date and time formulas is uploaded with CSV Import and mapped onto custom fields in Omeka, which are then mapped onto after_date and before_date in Neatline.

I hope that makes sense.

**Comment by sheepeeh** [comment]
+1 for the option to import tags.

**Comment by davidmcclure** [comment]
Yep, I agree. It would be easy to apply Omeka's tagging system to the Neatine record model (I think we could just instantiate a generic model mixin provided by Omeka), but there would be a bit of work to make other parts of Neatline understand the Omeka-style tags. Eg, the [stylesheet system](http://docs.neatline.org/neatline-stylesheets.html) in the editor uses tags as the "hooks" that it latches onto when it applies style rules, and we'd have to do some pretty significant refactoring to migrate over that functionality.

**Comment by omegader** [comment]
Using the imported tags to style the records is exactly what I'm after. I figured if the Omeka tags were created to spec (at least four characters, no hyphens, etc.), it would not be a problem. But people don't always think ahead. So maybe some kind of disclaimer next to a radio button?

**Comment by davidmcclure** [comment]
Oh, I see what you're saying @omegader - basically just _copy_ over the Omeka tags into the plaintext Neatline format when an item is linked. I hadn't thought of that - would be super easy.

**Comment by omegader** [comment]
Yep. That way you don't have to tag items twice.

**Comment by EvanPHamilton** [comment]
Just bumping that this would be very helpful!

**Comment by sgbalogh** [comment]
Just wanted to check if there is still interest in implementing this simple tag copying feature from Omeka to Neatline? I'm just about to start trying it manually with SQL update commands, but this would be a lovely feature!

**Comment by waynegraham** [comment]
We're starting to prioritize features for a new release and I'd love to have this in a release in the not-to-distant future.

**Comment by sylviachong** [comment]
Not sure if I should post this here or start a new thread, but in addition to importing tags from Omeka records, it would be REALLY useful to also import dates. I've spend an inordinate amount of time creating Omeka records for a timeline (that I can port to a new Omeka build, and which can be combined into different timelines). The records all have tags, and start (and some end) dates, but those are not importing into Neatline, and it's quite time consuming to have to hand-enter them for hundreds of records. (FYI I'm using Neatline Time for now to make timelines that I can import into Omeka exhibits, but I'd rather have them live in a Neatline exhibit, but right now the limitations of the interface, esp. re: time-consuming copy-and-pasting of metadata I'd like to use across records and exhibits, is preventing me from working in Neatline.)

**Comment by erochest** [comment]
Hi @sylviachong , thanks for the feature request. This does seem pretty different to me (not conceptually, but both the use cases and the mechanisms to implement it are). I am going to go ahead and move this to a new issue, and tag you so that you can keep track of it.

It's now at #371
